documentation/src/main/asciidoc/topics/ref_jgroups_system_properties.adoc

```diff
@@ -100,15 +100,15 @@ The following system properties apply to `default-jgroups-kubernetes.xml`:
 | Required/Optional
 
 | `jgroups.dns.query`
-| Sets the DNS record that returns cluster members.
-| N/A
-| Required if you do not set the `dns_query` parameter.
+| Sets the DNS record that returns cluster members and overrides the `dns_query` parameter in the DNS_PING discovery protocol.
```

Comment:
I would remove `dns_query` from the table and make the attribute required.
To overwrite `dns_query` you need a custom stack and, if that is the case, `jgroups.dns.query` is not longer used.